packages/grid/src/HeaderWrapper.tsx

```diff
@@ -115,15 +120,16 @@ export const HeaderWrapper: React.FC<IHeaderWrapper> = ({
     setChangingColumns('move');
   };
 
-
   return (
     <Header ref={ headerRef } width={ fullWidth } translateX={ translateX } theme={ theme }>
       { mappedColumnsRef.current.map((el: IColumn, index: number) => (
         el.visible
         && (
           <HeaderCellWrapper
+            sortable={ el.sortable }
+            sortOrder={ el.sortOrder }
             key={ el.field }
-            isEmpty={ index === emptyColumnIndexRef.current }
+            isEmpty={ isMoving && index === emptyColumnIndexRef.current }
```

Comment:
Could you explain please why you add this logic?
It worked incorrect?

Reply:
yes, bcs need 'isEmpty' only if moving